src/lib/styles/material/_table.scss

```diff
@@ -110,7 +110,6 @@
         &:hover
         {
             background: var(--color-structure-2);
-            cursor: pointer;
```

Comment:
Ich glaube die Task sah noch vor, den Pointer anzuzeigen, wenn ein routerLink drauf ist, oder!?

Reply:
So wie ich das verstanden hab war das ein Beispiel und sagt aus, dass es einfach implementierbar sein sollte. Daher hab ich das erstmal defaultmäßig raus genommen und wer es braucht kann es in der Komponente in Terra einfach zufügen.

Reply:
Kann das aber noch anpassen wenn das gewünscht ist. Ich hatte das in der PluginSetOverview mal implementiert und da war es einfach machbar

Reply:
Können wir gerne nochmal besprechen.
Ich denke halt, dass wenn ein routerLink drauf sitzt, die Tabellenzeile auf jeden Fall auf den Klick reagiert, und ein pointer angebracht wäre. Da macht man also höchstwahrscheinlich nichts mit falsch.